tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -251,6 +481,15 @@ def on_sync_object(self, data):
         )
         self._request_coll.on_map(sync_type, new_object)
         if new_object:
+            # Check and raise any alerts if required
+            #if sync_type.str == "health":
```

Comment:
This is kept intentionally as it could be enabled later for other alerts. May be would add a TODO there.